mirgecom/profiling.py

```diff
@@ -51,15 +52,68 @@ class SingleCallKernelProfile:
     footprint_bytes: int
 
 
+class StatisticsAccumulator:
+    """Class that provides statistical functions for profile results.
+
+    Values are converted to "Giga".
+    """
+
+    def __init__(self):
+        import sys
+        self.num_values = 0
+        self._sum = 0
+        self._min = sys.maxsize
+        self._max = 0
```

Comment:
I think it's better to start with `None`, to avoid assuming bounds.

Reply:
Done.

Reply:
Doesn't look done. Forgot to push?

Reply:
Yep, sorry. Should be pushed now.